src/interchange/toml/lexer.zig

```diff
@@ -207,7 +225,6 @@ pub const Lexer = struct {
 
         var is_legacy_octal_literal = false;
 
-        // Assume this is a number, but potentially change to a date/time later;
         lexer.token = T.t_numeric_literal;
```

Comment:
You may want to revert this to minimize the diff

Reply:
Ah fair, at first, my changes made it invalid